app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -553,7 +554,8 @@ fun CreateActivityScreen(
                       Toast.makeText(context, "You must select a location.", Toast.LENGTH_SHORT)
                           .show()
                     } else {
-                      attendees += profileViewModel.userState.value!!
+                      if (selectedOptionType == ActivityType.INDIVIDUAL.name)
+                          attendees += profileViewModel.userState.value!!
```

Comment:
Credit to @hmaryam84 for this suggestion :

```suggestion
                          attendees += profileViewModel.userState.value ?: emptyList()
```

Reply:
You can directly accept the suggestion from here, and do a commit from GitHub.